src/index.html

```diff
@@ -83,32 +82,38 @@ <h2>Tools</h2>
   <ul>
     <li><a href="https://tschaub.net/gpq/">Browser-based converter</a>: powered by the GPQ library, you can convert GeoJSON to
       GeoParquet and vice-versa, from within your browser.
-    <li><a href="https://geopandas.org/en/stable/docs/user_guide/io.html#apache-parquet-and-feather-file-formats">GeoPandas</a> (Python)</li>
-    <li><a href="https://qgis.org">QGIS</a> Windows installer now ships with GeoParquet support. Other versions work with GeoParquet as
-      long as they have a recent GDAL built with GeoParquet and its drivers.
+    <li><a href="https://geopandas.org/en/stable/docs/user_guide/io.html#apache-parquet-and-feather-file-formats">GeoPandas</a> (Python) extends the datatypes 
+      used by <a href="https://pandas.pydata.org/">pandas</a> to allow spatial operations on geometric types and supports <a href="https://geopandas.org/en/stable/docs/reference/api/geopandas.read_parquet.html">reading</a> and <a href="https://geopandas.org/en/stable/docs/reference/api/geopandas.read_parquet.html">writing</a> GeoParquet.</li>
```

Comment:
I find it a bit strange that GeoPandas is being mentioned here under "Tools", while it is very similar as other libraries mentioned below in a separate list?